etna/datasets/utils.py

```diff
@@ -271,3 +271,28 @@ def get_level_dataframe(
     target_level_df = pd.DataFrame(data=target_level_data, index=df.index, columns=target_level_segments)
 
     return target_level_df
+
+
+def inverse_transform_target_components(
```

Comment:
Do we really need this function to be public?

Reply:
Yes, this function might be useful and it is strange to have private method in utils